File: app/src/main/java/uk/ac/swansea/codenames/socketConnection.java
package uk.ac.swansea.codenames;

import java.net.URI;

import io.socket.client.IO;
import io.socket.client.Socket;

public class socketConnection {
    public static Socket socket = IO.socket(URI.create("http://192.168.1.124:3000"));;

    public socketConnection() {
    }
}

File: app/src/main/java/uk/ac/swansea/codenames/SocketConnection.kt
package uk.ac.swansea.codenames

import io.socket.client.IO
import io.socket.client.Socket
import java.net.URI

/**
 * Connects the client to the server.
 */
object SocketConnection {
    @JvmField
    //The server host I use.
    var socket: Socket = IO.socket(URI.create("https://codenamesserver-340711.nw.r.appspot.com"))
}